packages/editor/src/browser/editor-contribution.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2017 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { EditorManager } from './editor-manager';
+import { injectable, inject } from "inversify";
+import { StatusBarAlignment, StatusBar } from '@theia/core/lib/browser/status-bar/status-bar';
+import { FileIconProvider } from '@theia/filesystem/lib/browser/icons/file-icons';
+import { Position } from 'vscode-languageserver-types';
+import { FrontendApplicationContribution } from '@theia/core/lib/browser';
+import { Languages } from '@theia/languages/lib/common';
```

Comment:
You should explicitly add the dependency from editor extension to languages, otherwise it won’t work outside of theia repo

Reply:
done